frontend/event/eventController.js

```diff
@@ -353,25 +363,26 @@
            }
         }
 
+        function initUrlFields() {
```

Comment:
Why it has a verification for a undefined event?

Reply:
If it has an event, the fields should are the existent URLs in the event and empty URLs in the end. Otherwise the fields should are only empty URLs.